client/src/main/java/org/spine3/base/StringifierRegistry.java

```diff
@@ -85,7 +84,7 @@ private StringifierRegistry() {
     /**
      * Obtains a {@code Stringifier} for the passed type.
      *
-     * @param <T> the type of the values to convert
+     * @param <T>     the type of the values to convert
```

Comment:
Please make the ordering of `@param` consistent: first you describe the method arguments, then you describe the generic args. 

It's not only here. Please review other similar cases within this PR.